pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -213,6 +217,10 @@ def __load_weights_on_main_process(self) -> None:
 
         self._model = model
 
+    def _close_logger(self, trainer) -> None:
+        if hasattr(trainer, "logger"):
+            trainer.logger.close()
```

Comment:
Not all loggers define `close`, this will fail if not tensorboard. Why not just use `finalize`?